pkgs/development/tools/misc/grcov/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, rustPlatform, fetchFromGitHub, llvmPackages }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "grcov";
+  version = "v0.8.7";
+
+  src = fetchFromGitHub {
+    owner = "mozilla";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-4McF9tLIjDCftyGI29pm/LnTUBVWG+pY5z+mGFKySQM=";
+  };
+
+  cargoSha256 = "sha256-CdsTAMiEK8udRoD9PQSC0jQbPkdynFL0Tdw5aAZUmsM=";
+
+  # tests do not find grcov path correctly
+  checkFlags = let
+    skipList = [
+    "test_coveralls_service_job_id_is_not_sufficient"
+    "test_coveralls_service_name_is_not_sufficient"
+    "test_coveralls_works_with_just_service_name_and_job_id_args"
+    "test_coveralls_works_with_just_token_arg"
+    "test_integration"
+    "test_integration_guess_single_file"
+    "test_integration_zip_dir"
+    "test_integration_zip_zip"
+    ];
+  skipFlag = test: "--skip " + test;
+  in builtins.concatStringsSep " " (builtins.map skipFlag skipList);
+
+  meta = with lib; {
+    maintainers = with maintainers; [ AndersonTorres DieracDelta ];
```

Comment:
I'm not so sure to handle Rust packages now. 
Do you think on maintaining this package here in Nixpkgs?

Reply:
Sure! Would love to.